tensorboard/webapp/widgets/card_fob/card_fob_types.ts

```diff
@@ -13,9 +13,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 /**
- * The start and end time to define a linked time.
+ * The start and end time to define a time selection.
```

Comment:
Maybe mention which features its used for - linked time and step selector? Maybe. Up to you.

Reply:
Added